src/datasource.ts

```diff
@@ -382,14 +385,19 @@ export default class IrondbDatasource extends DataSourceApi<IrondbQueryInterface
               data.evalMatches = [];
               data.evalMatches.push(alert_match);
 
+              let notes =
+                rule !== undefined && rule !== null && rule['notes'] !== null && rule['notes'] !== ''
+                  ? rule['notes']
+                  : 'Oh no!';
```

Comment:
Note should be empty if the ruleset's notes were empty.  This sounds like debugging code.